src/Microsoft.VisualStudio.LanguageServices.Razor/OOPTagHelperResolverFactory.cs

```diff
@@ -16,36 +12,10 @@ internal class OOPTagHelperResolverFactory : ILanguageServiceFactory
     {
         public ILanguageService CreateLanguageService(HostLanguageServices languageServices)
         {
-            if (!IsRemoteClientWorking())
```

Comment:
@rynowak this was my primary reason for this PR. Figured it made sense to remove but wanted to make sure you didn't have any other meaning behind this bit.

Reply:
Yes, this was just a temporary workaround.